Wrapping/R/CMakeLists.txt

```diff
@@ -30,6 +30,10 @@ set_source_files_properties(${swig_generated_file_fullname} PROPERTIES COMPILE_F
 # set the output directory for the R library to the binary packaging location
 set_target_properties( ${SWIG_MODULE_SimpleITK_TARGET_NAME} PROPERTIES LIBRARY_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/Packaging/SimpleITK/src/)
 
+if (WIN32)
+	set_target_properties( ${SWIG_MODULE_SimpleITK_TARGET_NAME} PROPERTIES LINK_FLAGS "-s -shared -static-libgcc -static-libstdc++ -static -lpthread" )
+endif()
```

Comment:
I've had no luck setting these flags from the Superbuild for simpleitk. Any suggestions?